package.json

```diff
@@ -47,13 +46,16 @@
     "diff": "^2.2.1",
     "doctoc": "^1.0.0",
     "duplexer": "~0.1.1",
+    "generate-function": "^2.0.0",
```

Comment:
why was this library included? I don't see any `require('generate-function')`s within the code?